### PR TITLE
[WFLY-6531] Unsupported is not a valid module classification upstream…

### PR DESCRIPTION
**Pull request by kabir**
…. Use private instead

https://issues.jboss.org/browse/WFLY-6531

### CODE REVIEW

**Comment by kabir** [comment]
@bstansberry The undertow file conflicted. You had a previous commit _removing_ the module classification totally. My PR makes these two modules private. If you are right, we should close this PR and remove the module classification from the azure module. If I am right, this PR can be merged.

**Comment by bstansberry** [comment]
@kabir I'm right re: io.undertwo.js. See https://mojo.redhat.com/docs/DOC-1053168. And unless @pferraro says otherwise, I see no reason why org.jgroups.azure would be different from org.jgroups, which is also covered in that doc.

If EAP converts a module to unsupported when they really meant it should be private in upstream, that's a bug in EAP, as the meaning of unsupported is basically "public in upstream but Red Hat doesn't support direct use in our products." The only difference in EAP's formal definitions of "private" and "unsupported" is that the definition of "unsupported" includes an extra sentence saying that the module is public in upstream.

**Comment by bstansberry** [comment]
My last comment was slightly wrong in details. See https://mojo.redhat.com/docs/DOC-125270. The difference between "private" and "unsupported" is "private" includes the extra sentence "Marked "private" in the upstream community code base."  Same logical effect as what I said in my last comment though.

**Comment by pferraro** [comment]
@bstansberry org.jgroups is meant to be public in WF.  org.jgroups.azure is a dependency of org.jgroups and thus should share the same visibility.

File: feature-pack/src/main/resources/modules/system/layers/base/org/jgroups/azure/main/module.xml [hunk 1]
```diff
@@ -25,7 +25,7 @@
 <module xmlns="urn:jboss:module:1.3" name="org.jgroups.azure">
 
     <properties>
-        <property name="jboss.api" value="unsupported"/>
+        <property name="jboss.api" value="private"/>
```
Comment (rhusar) [inline feature-pack/src/main/resources/modules/system/layers/base/org/jgroups/azure/main/module.xml]: Actually, we would want this public then.

**Comment by kabir** [comment]
@bstansberry @pferraro so I make both public?

**Comment by bstansberry** [comment]
@kabir yes

**Comment by bstansberry** [comment]
Superceded by #8963

**Pull request closed without merging**